core/src/main/scala/org/elasticmq/actor/queue/QueueActorMessageOps.scala

```diff
@@ -177,6 +186,7 @@ trait QueueActorMessageOps extends Logging {
     messageQueue.byId.get(msgId).foreach { msgData =>
       if (msgData.deliveryReceipts.lastOption.contains(deliveryReceipt.receipt)) {
         // Just removing the msg from the map. The msg will be removed from the queue when trying to receive it.
+        inflightMessagesRegisty -= msgId
```

Comment:
[typo] Registy. Though maybe simply name it sth like `inflightMessageIds`, as that's what this is

Reply:
Great suggestion! Renamed